docker/Dockerfile

```diff
@@ -65,14 +65,14 @@ RUN \
     # from scratch (we have used 3.20.1, the last version by the time being). Once CentOS8
     # upgrade cmake to a version beyond 3.18.2, we can probably remove this hack and rely again in
     # yum-based installation
-    rpm -e cmake cmake-data cmake-filesystem cmake-rpm-macros && \
-    cd /opt && \
-    curl -OL https://github.com/Kitware/CMake/releases/download/v3.20.1/cmake-3.20.1.tar.gz && \
-    tar xvf cmake-3.20.1.tar.gz && \
-    cd cmake-3.20.1 && \
-    ./bootstrap && \
-    make && \
-    make install && \
+    #rpm -e cmake cmake-data cmake-filesystem cmake-rpm-macros && \
+    #cd /opt && \
+    #curl -OL https://github.com/Kitware/CMake/releases/download/v3.20.1/cmake-3.20.1.tar.gz && \
+    #tar xvf cmake-3.20.1.tar.gz && \
+    #cd cmake-3.20.1 && \
+    #./bootstrap && \
+    #make && \
+    #make install && \
```

Comment:
Not sure which cmake version comes with Centos 8.4.2105... Let's try and see.

Reply:
It seems it has worked!

I'll remove the comments blocks in the final version of this PR

NTC